### ISSUE TITLE
autoencoder的loss里重建误差l2范数有问题

### ISSUE THREAD

**Issue by LinkD3lta** [yj-zhou/Feature_Encoding_with_AutoEncoders_for_Weakly-supervised_Anomaly_Detection#1]
```
def dev_network_d(input_shape,modelname,testflag)函数里，这里
        sub_result = Subtract()([x_input, de2])
        cal_norm2 = Lambda(lambda x: tf.norm(x,ord = 2,axis=1))
        sub_norm2 = cal_norm2(sub_result)
        sub_norm2 = Reshape((1,))(sub_norm2)
        division = Lambda(lambda x:tf.div(x[0],x[1]))
        sub_result = division([sub_result,sub_norm2])
这段代码里sub_ressult已经是归一化后的重构误差（除以了l2范数），但在multi_loss这里，
    def multi_loss(y_true,y_pred):
        confidence_margin = 5. 
        dev = y_pred
        inlier_loss = K.abs(dev)
        outlier_loss = K.abs(K.maximum(confidence_margin - dev, 0.))

        sub_nor = tf.norm(sub_result,ord = 2,axis=1) 
        outlier_sub_loss = K.abs(K.maximum(confidence_margin - sub_nor, 0.))
        loss1 =  (1 - y_true) * (inlier_loss+sub_nor) + y_true * (outlier_loss+outlier_sub_loss)

        return loss1， sub_nor = tf.norm(sub_result,ord = 2,axis=1) 对sub_result这里又计算了l2范数，对归一化后的sub_result，已经是单位向量了，再次计算l2范数 不是恒为1吗？
